### PR TITLE
Kuguk added Google search tests

### PR DESCRIPTION
**Pull request by toleksyn**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-chrome-driver</artifactId>
             <scope>test</scope>
-            <version>${chrome.driver.version}</version>
+            <version>${selenium.java.version}</version>
```
Comment (toleksyn) [inline pom.xml:36]: change version to needed in chrome.driver.version

File: src/main/java/com/softserveinc/ita/kuguk/GoogleHomePage.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+
+import com.softserveinc.ita.pageobjects_task.kuguk.GoogleTest;
+import com.softserveinc.ita.pageobjects_task.kuguk.TestRunner;
+
+public class GoogleHomePage {
+
+	private By searchField = By.xpath("//input[@name='q']");
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/kuguk/GoogleHomePage.java:12]: move to method if only used once

**Review by kuguk — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (kuguk) [reply pom.xml:36]: fixed

File: src/main/java/com/softserveinc/ita/kuguk/GoogleHomePage.java (hunk 2 above)
Comment (kuguk) [reply src/main/java/com/softserveinc/ita/kuguk/GoogleHomePage.java:12]: fixed

**Pull request merged**